src/components/base/outline-google-map/outline-google-map.stories.ts

```diff
@@ -0,0 +1,202 @@
+import { html, TemplateResult } from 'lit-html';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { argTypeSlotContent } from '../../base/outline-element/utils/utils';
+import './outline-google-map';
+import './src/outline-map-marker';
+
+import { MapType, GestureType } from './outline-google-map';
+const mapTypeOptions: MapType[] = ['roadmap', 'satellite', 'hybrid', 'terrain'];
+const gestureOptions: GestureType[] = ['auto', 'cooperative', 'greedy', 'none'];
+
+export default {
+  title: 'Content/Map',
+  component: 'outline-google-map',
+  argTypes: {
+    ...argTypeSlotContent,
```

Comment:
Can we create a Storybook argType / arg for the `markers` slot? Right now it shows as `Default Slot` which seems misleading.